Makefile

```diff
@@ -178,37 +143,62 @@ include build/makelib/golang.mk
 
 RELEASE_VERSION=$(VERSION)
 RELEASE_BIN_DIR=$(BIN_DIR)
-RELEASE_CLIENT_SERVER_PLATFORMS=$(CLIENT_SERVER_PLATFORMS)
-RELEASE_CLIENT_ONLY_PLATFORMS=$(CLIENT_ONLY_PLATFORMS)
```

Comment:
we do however, the client vs server determination now happens in the release scripts